lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'takeaway'
+
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = { "Prawn Crackers" => 1.51, "Prawn Gyoza" => 3.51,
```

Comment:
As you mentioned in your own review, it would be nice to read these in from a file.  You could also consider creating a MenuItem class (and injecting the dependency).  One advantage would be that you could interact with menu items in a more readable way.  For example...

```ruby
# you could have a method like this on your order class to get the total price
def total
  items.sum(&:price)
end
```